R/bcdc_search.R

```diff
@@ -236,3 +236,24 @@ as.bcdc_recordlist <- function(x) {
   class(x) <- "bcdc_recordlist"
   x
 }
+
+
+#' Provide metadata for all the resources from a single B.C. Data Catalogue record
+#'
+#' @param x a list containing the metadata for a record
+#'  (result of running bcdc_get_record() function)
+#'
+#'
+#' @return A data frame containing the metadata for all the resources for a record
+#' @export
+#'
+#' @examples
+#' \donttest{
+#' airports <- bcdc_get_record("bc-airports")
+#' bcdc_tidy_resources(airports)
+#' }
+bcdc_tidy_resources <- function(x) {
+  df <- x$resource_df
```

Comment:
```suggestion
  if (!inherits(x, "bcdc_record")) stop("x needs to be generated from bcdc_get_record")
  x$resource_df
```

Reply:
I don't think we need to assign it

Reply:
> I think to be consistent, this argument should be called `record`. Happy to rename, however the current language comes straight from `bcdc_get_record` documentation, FWIW, and ☝️ suggestion is for `bcdc_record`. Let's decide and I can update both.

> I like the message - I wonder if it could be like the record name/id message and only appear once per session?
I think this is the kind of message that should be there every time users see the print method in the console? e.g. as you are looking at truncated df and see how to get it.